tools/perf/benchmark.common.js

```diff
@@ -0,0 +1,79 @@
+
+function runTests(env) {
+  const testData = JSON.parse(env.readFile("data.json"));
+
+  const results = {};
+
+  for (let testName in testData) {
+    let test = testData[testName];
```

Comment:
Would it make sense to split `data.json` into multiple JSONs, one for each benchmark? `simple.ftl` would go with `simple.json` etc. I think it would make it clearer what `data.json` actually is, and also easier to find out which variables are available in each benchmark.

Reply:
Maybe something like I suggested in https://github.com/projectfluent/fluent/issues/249 would work with that well? I'd prefer to make such changes as part of the reference fixtures, if that's ok with you.

Reply:
I think that's a larger task, really. Splitting the jsons sounds simple enough that I'd be in favor of doing it in this PR.

Reply:
Adding this here is also not trivial. I need to find per-engine way of reading the content of a directory to collect the list of tests available. I'll try to see how much work it is

Reply:
Ah, I see what you mean. This is both the storage of variable values _and_ the list of tests to run. I missed that before. Maybe renaming this file to `tests.json` would already be an improvement?

Reply:
Or, `benchmarks.json`, or `testsuites.json`.. you get the idea. `data` is a bit cryptic :)